client/src/App.js

```diff
@@ -32,6 +33,7 @@ function App() {
           <Switch>
             <Route path="/signup" component={SignUp} />
             <Route path="/login" component={Login} />
+            <Route path="/review" component={review} />
```

Comment:
I think this should be a protected route

Reply:
This review route is not the same route someone written previously for Review page.  it's just for testing reasons and will be deleted before merge.

Reply:
On this note, can we make this dialog appear on top of any page when the "Upload Code" button is pressed on the Navbar?